website/docs/docs/deploy/ci-jobs.md

```diff
@@ -77,6 +76,15 @@ If you're not using dbt Cloud’s native Git integration with [GitHub](/docs/cl
       - `non_native_pull_request_id` (for example, BitBucket)
    - Provide the `git_sha` or `git_branch` to target the correct commit or branch to run the job against. 
 
+## Test metrics in CI <Lifecycle status="team,enterprise" />
+Automatically test your metric definition models (metrics, semantic models, and saved queries) during code reviews by running warehouse validation checks in your CI job.
+- Testing metric definitions in a CI job guarantees that any code changes made to dbt models don't break these metrics. 
+- It allows you to catch issues early in the development process and deliver high-quality data to your end users.
+- To enable metrics in CI, add the `dbt sl validate` command in the **Execution settings** section on the **Job settings** page (explained in [Step 2](/docs/deploy/ci-jobs#prerequisites) earlier). Make sure you've set up (or are setting up) your CI job first before adding this command.
```

Comment:
Can we update this to `dbt sl validate --select state:modified+`. This is our recommended approach so I want it to be front and center in the docs.